app/models/application_record.rb

```diff
@@ -1,3 +1,3 @@
-class ApplicationRecord < ActiveRecord::Base
```

Comment:
Delete this unnecessary class, make all models inherit directly from ActiveRecord::Base